### PR TITLE
Add Bioschemas for chemical substances #BioHackEU20

### PR DESCRIPTION
**Pull request by egonw**
Currently, for two `P31`:

* JRC representative nanomaterial: Q47461491
* nanomaterial: Q967847

### CODE REVIEW

**Comment by egonw** [comment]
> @egonw. Did you fix the "console.log" issue?

@fnielsen, how do you want it fixed? Remove it from the catch? It make it the same as in the other catch clauses? Or remove if everywhere?

**Review by fnielsen — COMMENTED** [review]
> > @egonw. Did you fix the "console.log" issue?
> 
> @fnielsen, how do you want it fixed? Remove it from the catch? It make it the same as in the other catch clauses? Or remove if everywhere?



> > @egonw. Did you fix the "console.log" issue?
> 
> @fnielsen, how do you want it fixed? Remove it from the catch? It make it the same as in the other catch clauses? Or remove if everywhere?

Are there supposed to be console.log entries at all. Shouldn't they all be removed unless you are doing debugging?

File: scholia/app/templates/base.html [hunk 1]
```diff
@@ -82,7 +82,22 @@
 	 catch(e) {}
 	 
 	 /* BioSchemas annotation */
-	 if (item.claims.P225) {
+	 console.log(item.claims.P31[0].mainsnak.datavalue.value.id)
```
[the remark this answers is not in the document]
Comment (egonw) [reply scholia/app/templates/base.html]: Better now?

**Comment by fnielsen** [comment]
@egonw I am getting conflicts. And I am trying to understand the parents of the branch https://github.com/egonw/scholia/network Would it be possible for you to rebase?

**Comment by egonw** [comment]
> @egonw I am getting conflicts. And I am trying to understand the parents of the branch https://github.com/egonw/scholia/network Would it be possible for you to rebase?

done.

**Comment by fnielsen** [comment]
I am unsure which item demonstrate the use of the BioSchema.

**Comment by egonw** [comment]
@fnielsen, at least all of these: https://nanocommons.github.io/specifications/jrc/

**Pull request merged**